api/internal/crawl/crawler/github/crawler.go

```diff
@@ -129,7 +129,7 @@ func (gc githubCrawler) FetchDocument(ctx context.Context, d *doc.Document) erro
 			continue
 		}
 	}
-	return fmt.Errorf("file not found: %s", url)
+	return fmt.Errorf("file not found: %s, error: %v", url, err)
```

Comment:
in the future use errors.Wrap(err, "additional context for error")

Reply:
Got it! Thanks!